<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QWidget" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>856</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1286</width>
    <height>856</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1286</width>
    <height>856</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    border: none; /* 去掉边框 */
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
                                stop: 0 #1A1A1A,  /* 深黑色 */
                                stop: 1 #0D0D0D);  /* 更深的黑色 */
}

QLineEdit {
    background-color: #333;  /* 输入框背景颜色（深灰色） */
    border: 1px solid #666;  /* 输入框边框颜色（浅灰色） */
    border-radius: 5px;  /* 圆角边框 */
    padding: 5px;  /* 内边距 */
    font-size: 14px;  /* 字体大小 */
    color: #FFFFFF;  /* 字体颜色（白色） */
}

QPushButton {
    background-color: #444;  /* 按钮背景颜色（稍浅的黑色） */
    border: none;  /* 去掉边框 */
    border-radius: 5px;  /* 圆角边框 */
    color: #FFFFFF;  /* 字体颜色（白色） */
    font-size: 14px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}

QPushButton:hover {
    background-color: #555;  /* 悬停时的按钮背景颜色（灰色） */
}

QPushButton:pressed {
    background-color: #333;  /* 按下时的按钮背景颜色（更深的灰色） */
}

QLabel {
    color: #FFFFFF;  /* 标签字体颜色（白色） */
    font-size: 14px;  /* 标签字体大小 */
}
</string>
  </property>
  <widget class="QTextEdit" name="textEditAccept">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>-10</y>
     <width>941</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHistory">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>810</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonHistory {
    background-color: transparent;  /* 设置按钮背景为透明 */
    border: none;  /* 去掉边框 */
    color: rgb(0, 170, 255);  /* 设置字体颜色 */
    font-size: 30px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/lishijilu.png</normaloff>:/new/prefix1/lishijilu.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSend">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>940</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>940</width>
     <height>400</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>810</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonSend {
    background-color: transparent;  /* 设置按钮背景为透明 */
    border: none;  /* 去掉边框 */
    color: rgb(0, 170, 255);  /* 设置字体颜色 */
    font-size: 30px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/faosng.png</normaloff>:/new/prefix1/faosng.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonReg {
    background-color: transparent;  /* 设置按钮背景为透明 */
    border: none;  /* 去掉边框 */
    color: rgb(0, 170, 255);  /* 设置字体颜色 */
    font-size: 30px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/close1.png</normaloff>:/new/prefix1/close1.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>160</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/moren.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>道号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditName">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>380</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonClear {
    background-color: transparent;  /* 设置按钮背景为透明 */
    border: none;  /* 去掉边框 */
    color: rgb(0, 170, 255);  /* 设置字体颜色 */
    font-size: 30px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}
</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/clear.png</normaloff>:/new/prefix1/clear.png</iconset>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
  </widget>
  <zorder>textEditSend</zorder>
  <zorder>textEditAccept</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButtonHistory</zorder>
  <zorder>pushButtonSend</zorder>
  <zorder>pushButtonClose</zorder>
  <zorder>pushButtonClear</zorder>
  <zorder>lcdNumber</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
